lib/auth.jsx

```diff
@@ -124,4 +188,4 @@ const mapStateToProps = state => ( {
 	hasLoginError: hasLoginError( state ),
 } );
 
-export default connect( mapStateToProps )( Auth );
+export default connect( mapStateToProps, null, null, { pure: false } )( Auth );
```

Comment:
Oh! I think I misunderstood what it was for. I thought you needed it if doing local state. Will update.